range_related_properties.md

```diff
@@ -0,0 +1,148 @@
+
+## `slider` Role
+
+`aria-valuenow`, `aria-valuemin` and `aria-valuemax` are all required attributes for the `slider` role. `aria-valuetext` can be used when appropriate. Detailed description of the `slider` role can be found in the [slider design pattern](https://w3c.github.io/aria-practices/#slider) and [slider (multi-thumb) design pattern](https://w3c.github.io/aria-practices/#slidertwothumb).
+
+The following example is a temperature controller. `aria-valuetext` is not be used as the number value in `aria-valuenow` is meaningful to the user.
+
+```
+<div class="rail">
+  <div id="thumb" role="slider" aria-valuemin="50" aria-valuenow="68" aria-valuemax="100"
+       aria-label="Temperature (F)" tabindex="0">
+  </div>
+</div>
+
+```
+
+The slider example above can be made using the HTML input type=range element.
+
+```
+<input type="range" min="50" value="68" max="100" aria-label="Temperature (F)">
+```
+
+The following example is a fan control. The `aria-valuenow` value is "1", which is not meaningful to the user. The assistive technology will surface the value of `aria-valuetext` ("low") instead.
+
+```
+<div class="rail">
+  <div id="thumb" role="slider" aria-valuemin="0" aria-valuenow="1" aria-valuemax="3"
+       aria-valuetext="low" aria-label="Fan speed" tabindex="0" >
+  </div>
+  <div class="value"> Off </div>
+  <div class="value"> Low </div>
+  <div class="value"> Medium </div>
+  <div class="value"> High </div>
+</div>
+
+```
+
+## `spinbutton` Role
+
+`aria-valuenow`, `aria-valuemin` and `aria-valuemax` are all required attributes for the `spinbutton` role. `aria-valuetext` can be used when appropriate.
+
+This example sets the price of paperclips in cents.
+
+```
+<div role="spinbutton" aria-valuemin="1" aria-valuenow="50" aria-valuemax="200" tabindex="0">
+  <button id="lower-price">Lower</button>
+  <button id="raise-price">Raise</button>
+  Price per paperclip: $<span id="price">0.50</span>
+</div>
+```
+
+The slider example above can be made using the native HTML input type="number" element.
+
+```
+<input type="number" min="0.01" value="0.5" max="2" aria-labelledby="paperclip-label">
+<span id="paperclip-label">Price per paperclip</span>:
+$<output id="price" aria-labelledby="paperclip-label">0.50</output>
+```
+
+```
+<label>Price per paperclip: $<input type="number" min="0.01" value="0.5" max="2" step="0.01"></label>
+```
+
+## `progressbar` Role
+
+`aria-valuenow`, `aria-valuemin` and `aria-valuemax` are not required attributes for the `progressbar` role, however, but the attributes might be necessary for communicating the state of a progress bar to assistive technologies.
+
+This is an example of a progress bar represent by an SVG. The range properties are necessary to full explain the widget to assistive technologies.
+
+```
+<div for="loadstatus">Loading:
+  <span role="progressbar" id="loadstatus" aria-valuemin="0" aria-valuenow="33" aria-valuemax="100" >
```

Comment:
Maybe show how `aria-valuemin` and `aria-valuemax` can be omitted in this example, and that they have default values.